src/instructlab/model/list.py

```diff
@@ -34,32 +38,35 @@ def model_list(model_dirs: list[str], list_checkpoints: bool):
     """lists models"""
     data = []
     for directory in model_dirs:
-        for entry in Path(directory).iterdir():
-            # if file, just tally the size. This must be a GGUF.
-            if entry.is_file() and is_model_gguf(entry):
-                model, age, size = _analyze_gguf(entry)
-                data.append([model, age, size])
+        d = Path(directory)
+        if not d.is_dir():
+            continue
+        for entry in d.iterdir():
+            if is_model_gguf(entry):
+                data.append(_analyze_gguf(entry))
             elif entry.is_dir():
-                for m in _analyze_dir(entry, list_checkpoints, directory):
-                    data.append(m)
-    print_table(["Model Name", "Last Modified", "Size"], data)
+                data += list(_analyze_dir(entry, list_checkpoints, directory))
+    print_table(("Model Name", "Last Modified", "Size"), data)
 
 
-# convert_bytes_to_proper_mag takes a dir/file size in Bytes and converts it to the proper Magnitude (MiB, GiB)
-def convert_bytes_to_proper_mag(f_size: float):
-    magnitudes = ["KiB", "MiB", "GiB", "TiB"]
+def get_bytes_size_string(n_bytes: int) -> str:
```

Comment:
I think the newly proposed name is less clear. How about `convert_bytes_to_human_readable_size`? Something along those lines might be more descriptive.